git-ftp

```diff
@@ -752,8 +752,14 @@ submodule_catchup() {
 }
 
 set_submodule_args() {
-	# Duplicate the current required parameters
-	args=(--user "$REMOTE_USER" --passwd "$REMOTE_PASSWD")
+	# Duplicate the current required parameters if set
+	args="" 
+	if [ -z "$REMOTE_USER" ]; then
```

Comment:
Did you test this? If I understand it correctly, you are setting `args+=(--user "")` when `$REMOTE_USER` is an empty string.

Did you mean `-n`?
```bash
if [ -n "$REMOTE_USER" ]; then
    ...
fi
```